examples/misc/lib/tutorial/daily_news.dart

```diff
@@ -17,22 +17,22 @@ import 'dart:async';
 // #docregion sync
 // Synchronous code
 void _printDailyNewsDigestSync() {
-  var news = _gatherNewsReportsSync(); // Can take a while.
-  print(news);
+  var newsDigest = _gatherNewsReportsSync(); // Can take a while.
+  print(newsDigest);
 }
 // #enddocregion sync
 
 // #docregion main-async
 Future<void> _printDailyNewsDigestAsync() async {
-  var news = await _gatherNewsReportsAsync();
-  print(news);
+  var newsDigest = await _gatherNewsReportsAsync();
+  print(newsDigest);
 }
 // #enddocregion main-async
 
 // #docregion main-future-api
 Future<void> _printDailyNewsDigestAsyncUsingFutureAPI() {
   final future = _gatherNewsReportsAsync();
-  return future.then((news) => print(news));
+  return future.then((newsDigest) => print(newsDigest));
```

Comment:
Effective Dart [would tell you](https://www.dartlang.org/guides/language/effective-dart/usage#dont-create-a-lambda-when-a-tear-off-will-do) to just do:

```dart
return future.then(print);
```

Doing that is a little confusing in a tutorial, though. Maybe there's a way to do a little extra work in there to justify the lambda so it doesn't break the style guide?